docs/examples/One_to_One_Associator.py

```diff
@@ -0,0 +1,358 @@
+"""
+Generic One-to-One Association Examples
```

Comment:
Other 1-2-1 association algorithms can only associate tracks or states. This is generic because it can associate anything with a valid measure function.